server/methods/core/orders.js

```diff
@@ -324,6 +343,71 @@ Meteor.methods({
    */
   "orders/sendNotification": function (order) {
     check(order, Object);
+    const shoppersPhone = order.billing[0].address.phone;
+    Logger.info("CUSTOMER ORDER DETAILS", order.items);
+    Logger.info("CUSTOMER'S EMAIL", order.email);
+    Logger.info("CUSTOMERS PHONE NO " + shoppersPhone);
+
+    let vendorPhones = [];
```

Comment:
array vendorPhones was re-assigned, I think using const will make the size of the array to be fixed and hope the const won't affect the array filtering I later did.